emulators/dosbox-staging/Portfile

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem            1.0
+PortGroup             github 1.0
+PortGroup             meson 1.0
+
+github.setup          dosbox-staging dosbox-staging 0.78.1 v
```

Comment:
please add `github.tarball_from archives` and recalculate the `checksums`

Reply:
What is the difference between archives and using source code from releases?